### PR TITLE
IDF F-16 Sufa Mod support

### PR DESCRIPTION
**Pull request by kevotosmart**

### CODE REVIEW

File: gen/flights/ai_flight_planner_db.py [hunk 1]
```diff
@@ -130,6 +131,7 @@
 
 CAP_CAPABLE = [
     Su_57,
+    F_16I,
```
Comment (DanAlbert) [inline gen/flights/ai_flight_planner_db.py]: This is better at CAP than the F-22? I don't buy that.

Comment (kevotosmart) [reply gen/flights/ai_flight_planner_db.py]: I’m sorry, does py prioritize the list from top being a more preferred CAP plane? If so can fix it and put it where it really belongs.

Comment (kevotosmart) [reply gen/flights/ai_flight_planner_db.py]: I put it where it belongs

**Comment by kevotosmart** [comment]
Mods on version 1.2


https://discord.com/invite/YDUecPFNAU

**Comment by kevotosmart** [comment]
Found a conflicting issue.

**Pull request closed without merging**